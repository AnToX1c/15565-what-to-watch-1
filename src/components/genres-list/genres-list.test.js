import React from "react";
import renderer from "react-test-renderer";
import GenresList from "./genres-list";

const mock = [`All genres`, `Thrillers`, `Dramas`, `Kids & Family`, `Comedies`];

it(`GenresList page correctly renders`, () => {
  const tree = renderer
    .create(
        <GenresList genres={mock} onClick={jest.fn()} activeGenre={mock[0]} />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
