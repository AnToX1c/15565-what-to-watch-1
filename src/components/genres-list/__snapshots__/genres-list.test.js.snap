// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenresList page correctly renders 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Thrillers
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Dramas
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Kids & Family
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Comedies
    </a>
  </li>
</ul>
`;
